Week-4/Excercise 4.2/InheritenceExample.js

```diff
@@ -0,0 +1,31 @@
+const Person = function () {};
+
+// initializing function using .prototype to set name and age properties
+Person.prototype.initialze = function (name, age) {
+  this.name = name;
+  this.age = age;
+};
+
+const Teacher = function () {};
+
+// Inhert from Person prototype
+Teacher.prototype = Object.create(Person.prototype);
+// setting the constructor to Teacher
+Teacher.prototype.constructor = Teacher;
+// initializing function using .prototype to set name and age properties
```

Comment:
FYI u can also implement it using setPrototypeOf

Reply:
@anilrao316, okay I will check it out, thanks.